main.go

```diff
@@ -112,3 +119,15 @@ Arguments:
 		helpTab,
 	)
 }
+
+//This section translates a windows path such as "C:\Users\user" to "\Users\user"
+//AND removes the default paths for mingw and git4windows to fix specifying a file path breaking
+func gitbashWindowsDir(dir string) (res string) {
+	res = filepath.ToSlash(dir)
+	res = strings.Replace(res, "C:", "", -1)
+
+	// If you dont use "C:\mingw64" as the location where you installed mingw, copy this and replace /mingw64 with your install path
```

Comment:
Is there no way to detect this?

Reply:
It is possible. I just dont know how.

Reply:
Check the environment inside of mingw for something that could help. Maybe there'll be an env var called `MINGW_HOME` or something useful like that.

Reply:
I have looked. The only cygwin based environment that does this is
MobaXterm but it's a pile of crap... 26.33 minutes to install git...

Mingw, 5 tops, and that was with a handicap on my slowest USB I own.

Reply:
For example. this is the entire output of the `env` command without my custom enviroment variables in my `.bashrc` file
https://pastebin.com/Q7kyvCWp

Reply:
https://pastebin.com/Q7kyvCWp
Default Enviroment in my setup

Reply:
Found a possible way to detect this. Since telling the server to open a path like `/opt` will try and open `<Mingw install path>/opt`,

WE COULD detect what path `/` is on Mingw, and then lop that off.

Unfortunately I woke up with a headache today and that's the most I can do till the coffee kicks in.

Reply:
OKay. So i have come up with a slightly jank solution, Working and getting it to work

Basically this would only work for mingw, as git4win will usually be installed in default locations

this is some psudo code, moslty actual code. BUT IT WONT WORK
```go
package main

import (
	"fmt"
	"log"
	"os"
	"path/filepath"
)

var home = os.Getenv("HOME")
var installDir = home/../../
 var err error
// this is the part that doesnt work. It will go up two directories, 
//from /home/user to / and then return the absolute path for 
//that, that is what we cut off

func main() {
	installDir, err = filepath.Abs(installDir)
	if err != nil {
		log.Fatal(err)
	}
	fmt.Println("Install Dir:", installDir)
}
```

@eargollo i think i could use your help here